modules/installation-vsphere-machines.adoc

```diff
@@ -6,13 +6,14 @@
 = Creating {op-system-first} machines in vSphere
 
 Before you install a user provisioned infrastructure (UPI) cluster on vSphere,
-you need to create {op-system} machines on a single host for it to use.
+you need to create {op-system} machines for it to use.
 
 .Prerequisites
 
 * Obtain the Ignition config files for your cluster.
 * Have access to an HTTP server that you can access from your computer and that
 the machines that you create can access.
+* Create a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
```

Comment:
@staebler, since users are supposed to create a vSphere cluster, should we include a link to the instructions?